### PR TITLE
Small Embed Fixes

### PR DESCRIPTION
**Pull request by hariombalhara**
Fixes points 1 and 3 in #7419

### CODE REVIEW

File: packages/embeds/embed-core/src/embed-iframe.ts [hunk 1]
```diff
@@ -340,8 +340,15 @@ function keepParentInformedAboutDimensionChanges() {
       throw new Error("Main element should be an HTMLElement");
     }
 
-    const contentHeight = mainElement.offsetHeight;
-    const contentWidth = mainElement.offsetWidth;
+    const mainElementStyles = getComputedStyle(mainElement);
+    const contentHeight =
```
Comment (hariombalhara) [inline packages/embeds/embed-core/src/embed-iframe.ts:344]: offset doesn't include margin. So consider that as success page main element has margins

File: packages/embeds/embed-core/src/Inline/inlineHtml.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-const html = `<div id="wrapper" style="top:50%; left:50%" class="absolute z-highest">
-<div style="transform:translate(-50%,-50%)"  class="loader border-brand dark:border-darkmodebrand">
```
Comment (hariombalhara) [inline packages/embeds/embed-core/src/Inline/inlineHtml.ts:2]: style="top:50%; left:50%" and style="transform:translate(-50%,-50%)" has to be on same element for center positioning to work correctly.

**Review by hariombalhara — COMMENTED** [review]
Self review done

File: packages/embeds/embed-core/src/embed-iframe.ts (hunk 1 above)
Comment (sean-brydon) [reply packages/embeds/embed-core/src/embed-iframe.ts:344]: Oh the joys :')

**Review by sean-brydon — APPROVED** [review]
LGTM

**Pull request merged**